Parser/asdl_c.py

```diff
@@ -690,19 +695,50 @@ def visitModule(self, mod):
         }
         res = PyObject_SetAttr(self, name, PyTuple_GET_ITEM(args, i));
         Py_DECREF(name);
-        if (res < 0)
+        if (res < 0) {
             goto cleanup;
+        }
     }
     if (kw) {
         i = 0;  /* needed by PyDict_Next */
+        if (_PyObject_LookupAttrId(fields, &PyId_index, &index) < 0) {
+            goto cleanup;
+        }
```

Comment:
Is there a reason that why you didn't use `PySequence_Index` and manually accessed it?

Reply:
I didn't know `PySequence_Index` existed, TIL.